src/ui/layout/RecentObjectsList.vue

```diff
@@ -181,6 +181,8 @@ export default {
          */
         setSavedRecentItems() {
             localStorage.setItem(LOCAL_STORAGE_KEY__RECENT_OBJECTS, JSON.stringify(this.recents));
+            // send event to parent for enabled button
+            this.$emit("handleRecentObjectsListUpdated", this.recents);
```

Comment:
Two things:

- Let's only emit this if `this.recents.length === 1`. That will cut down on unnecessary emits, since we only care to enable the button if the list length becomes non-zero.
- Let's rename the event to "setClearButtonDisabled"
- Instead of emitting the list itself, let's just emit `false`.